src/cpp/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ file(GLOB_RECURSE SOURCE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp", "${CMAKE
 
 set(TARGET_NAME openvino_genai)
 add_library(${TARGET_NAME} SHARED ${SOURCE_FILES})
+add_dependencies(${TARGET_NAME} openvino_tokenizers)
```

Comment:
we have an option to build w/o tokenizers and copy prebuilt tokenizers library to genai library folder.

So, we need to check whether the `openvino_tokenizers` target exists first.

Reply:
resolved in https://github.com/Wovchena/openvino.genai-public/pull/68